From 997c5540841d79970835203f83d385baf3fb8f07 Mon Sep 17 00:00:00 2001
From: Dong Aisheng <aisheng.dong@nxp.com>
Date: Thu, 16 Mar 2017 13:13:34 +0800
Subject: [PATCH 1564/5242] MLK-14472-1 dts: imx6ul: add flexcan stop mode
 wakeup support

commit  74b0b8f0f8681e8fa93c9ed6958bed0acab626f1 from
https://source.codeaurora.org/external/imx/linux-imx.git

Add stop-mode property which is required by stop mode wakeup feature.

Signed-off-by: Dong Aisheng <aisheng.dong@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx6ul.dtsi |    1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/boot/dts/imx6ul.dtsi b/arch/arm/boot/dts/imx6ul.dtsi
index 1b709fd..d87f449 100644
--- a/arch/arm/boot/dts/imx6ul.dtsi
+++ b/arch/arm/boot/dts/imx6ul.dtsi
@@ -520,6 +520,7 @@
 				clocks = <&clks IMX6UL_CLK_CAN1_IPG>,
 					 <&clks IMX6UL_CLK_CAN1_SERIAL>;
 				clock-names = "ipg", "per";
+				stop-mode = <&gpr 0x10 1 0x10 17>;
 				status = "disabled";
 			};
 
-- 
1.7.9.5

